.rubocop.yml

```diff
@@ -89,6 +99,9 @@ Lint/HandleExceptions:
 Lint/UnusedBlockArgument:
   Enabled: false
 
+Lint/AmbiguousBlockAssociation:
```

Comment:
If this cop is enabled, autocorrect seems not to work.
Should we fix them manually?

![screen shot 2017-03-27 at 15 01 04](https://cloud.githubusercontent.com/assets/147051/24342846/4be490b8-12fe-11e7-9e8d-2366d02015fd.png)